docker/nginx/nginx.conf

```diff
@@ -63,4 +63,5 @@ http {
     #gzip  on;
 
     include /etc/nginx/conf.d/*.conf;
+    include /etc/nginx/conf.extra.d/*.conf;
```

Comment:
Can we call this `conf.accounts.d` instead of `conf.extra.d`? Just so we know the relationship in the future.